.gitignore

```
@@ -0,0 +1,6 @@
+.ipynb_checkpoints/
+__pycache__/
+.Rproj.user
+.RData
+.Rhistory
+.Ruserdata
```

```suggestion
.Ruserdata

```

Note the ["github suggestion"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes). i.e. you can directly commit the suggestion without having to do anything other than click a button.

All the suggestion is doing is to get rid of the pesky red symbol in line 6